### PR TITLE
Fix unclosed foreach

### LINKED ISSUE #26656: FO - An exception is displayed in the order-detail page after enabling merchandise return
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add screenshots

In the FO > order-history > order-detail page, an exception is displayed after enabling merchandise return.

This issue is detected by [Nightly](https://nightly.prestashop.com/report/2989)

There is no exception if we have an order with a vector product

### Expected behavior

Should be well displayed without any exception

### Steps to reproduce

1. Go to BO > Customer Service > Merchandise Returns page and enable returns
2. Go to BO > Orders > Orders page and change any order status to "Shipped"
3. Go to FO > order-history > Open the previous order-detail page
image

There is no exception if we have an order with a vector product

### PrestaShop version(s) where the bug happened

develop

### PHP version(s) where the bug happened

7.2

### If your bug is related to a module, specify its name and its version

_No response_

### PR DESCRIPTION
**Pull request by ghost**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | After i rebase this PR (https://github.com/PrestaShop/PrestaShop/pull/25204) i introduce a bug this PR fix the bad rebase
| Type?             | bug fix
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes https://github.com/PrestaShop/PrestaShop/issues/26656
| How to test?      | Check the error and check the part to test of this PR : https://github.com/PrestaShop/PrestaShop/pull/25204
| Possible impacts? | No

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/27052)

### CODE REVIEW

File: themes/classic/templates/customer/_partials/order-detail-return.tpl [hunk 1]
```diff
@@ -40,35 +40,23 @@
         {foreach from=$order.products item=product name=products}
           <tr>
             <td>
+              {if !$product.is_virtual}
               <span id="_desktop_product_line_{$product.id_order_detail}">
```
Comment (PierreRambaud) [inline themes/classic/templates/customer/_partials/order-detail-return.tpl]: Indentation is broken

**Comment by matks** [comment]
Hello, the CI on develop branch was failing today because of https://github.com/PrestaShop/PrestaShop/issues/27062

We fixed the issues on develop branch but you need to rebase to make CI pass successfully on your Pull Request

**Review by PierreRambaud — CHANGES_REQUESTED** [review]
PR needs to be rebased

**Review by matthieu-rolland — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Review by NeOMakinG — APPROVED** [review]

**Comment by hibatallahAouadni** [comment]
Hello @okom3pom 

Thanks for your PR! 
LGTM, QA :heavy_check_mark: 

https://user-images.githubusercontent.com/23308427/148601286-6f35d6bd-7a25-4905-ad49-9e5e09a504a9.mp4

Thanks!

**Comment by Progi1984** [comment]
Thanks @okom3pom & @hibatallahAouadni

**Pull request merged**